packages/shell-api/src/shell-log.ts

```diff
@@ -0,0 +1,54 @@
+import type { MongoshBus } from '@mongosh/types';
+
+export interface ShellLog {
```

Comment:
Should this extend `ShellApi`?

Reply:
You mean `ShellApiClass`? Yes, it probably should for consistency (and to ensure that we add e.g. `help` text for the new methods).

Reply:
Yeah, the same - must have gotten distracted halfway through typing it 😵‍💫

Reply:
This was my initial thought as well, but then I looked at the ShellBson and now I am wondering how it is different. Or should ShellBson also extend ShellApi?

Reply:
`ShellBson` groups a few classes/constructors together, but this one is a class that the user would interact with directly, so yes, they are a bit different.